package.json

```diff
@@ -76,12 +77,12 @@
   "dependencies": {
     "@aws-sdk/credential-providers": "^3.438.0",
     "@aws-sdk/lib-dynamodb": "3.363.0",
+    "@govuk-one-login/di-ipv-cri-common-express": "3.2.0",
```

Comment:
Thanks for doing this @merlinc! I guess we should give it a good regression test before merging since we were 2 major versions behind? 

cc @Z-Mohamoud